principal-prop-on-premise/README.md

```diff
@@ -0,0 +1,151 @@
+# Configuring Principal Propagation to an On-Premise System
+
+This sample provides details on how to configure principal propagation when you extend an on-premise system using SAP BTP, Kyma runtime.
+
+## Context
+
+The principal propagation relies on the exchange of the JWT token received in Kyma. The exchanged token is then forwarded to the SAP Cloud Connector and is used to identify the logged-in user.
```

Comment:
Is the token generated in the Kyma runtime? How? Is the Cloud Connector in the diagram the same SAP Cloud Connector mentioned here?